app/main/controllers/template/RTMediaTemplate.php

```diff
@@ -228,7 +228,8 @@ public function add_hidden_fields_in_gallery() {
 	public function check_return_json() {
 		global $rtmedia_query;
 
-		if ( 'json' === $rtmedia_query->format ) {
+		// Ensure $rtmedia_query is not null and has the 'format' property
+		if ( isset( $rtmedia_query->format ) && 'json' === $rtmedia_query->format ) {
```

Comment:
First check if $rtmedia_query is NULL.
Check this for all the other conditions.